src/main/java/se/complexjava/videostreamingapi/exceptionhandling/exception/ResourceCreationException.java

```diff
@@ -0,0 +1,14 @@
+package se.complexjava.videostreamingapi.exceptionhandling.exception;
+
+public class ResourceCreationException extends Exception {
+
+    public ResourceCreationException(String message) {
+
+        super(message);
+    }
```

Comment:
Jag har lite dålig koll på detta. Finns det fall när detta exception kastas och man ej får med cause? Samma fråga för övriga exceptions med samma utformning.

Reply:
man väljer själv om man vill lägga med cause. felhanteringen har jag inte gått in på så mycket utan tänker att det växer fram allt eftersom. fritt fram för alla att experimentera